ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/utils/StringUtils.java

```diff
@@ -162,4 +162,14 @@ public static String getErrorStringValue(Throwable error) {
         }
         return error.getMessage();
     }
+
+    /**
+     * Convert unicode character to ballerina unicode format.
+     *
+     * @param character unicode character to convert.
+     * @return reformatted unicode string.
```

Comment:
Use sentence case for these documentation as well.

Reply:
Updated in 2b06d029352f8a27893b7d41e10e3ad95aa851ea